osu.Game/Overlays/Mods/ModButtonEmpty.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Rulesets.Mods;
+
+namespace osu.Game.Overlays.Mods
+{
+    /// <summary>
+    /// A mod button used exclusively for providing an empty space the size of a mod button.
+    /// </summary>
+    public class ModButtonEmpty : Container

```

Comment:
please provide a suggestion. i think this is a step up from using `ModButtonBase` (i'm really against using the base suffix, even though we use it in a few places).